src/FunctionalComponent/FunctionalComponent.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+import './FunctionalComponent.css'
+
+const functionalComponent = () => {
```

Comment:
it's better to name components from a capital letter
```suggestion
const FunctionalComponent = () => {
```

Reply:
Used Capital letter for components name.